core/app/modules/kit-library/assets/js/components/taxonomies-filter.js

```diff
@@ -28,6 +28,43 @@ export default function TaxonomiesFilter( props ) {
 						taxonomiesByType={ group }
 						selected={ props.selected }
 						onSelect={ props.onSelect }
+						onOpen={ ( value, title ) => {
+							// TODO: Don't shoot this event on first load
+							if ( typeof ( value ) !== 'undefined' ) {
+								$e.run(
+									`kit-library/${ ( value ) ? 'expand' : 'collapse' }`,
+									{
+										section: title,
+										action: ( value ) ? 'expand' : 'collapse',
+										category: ( '/' === props.category ? 'all kits' : 'favorites' ),
+									},
+									{
+										meta: {
+											event: 'sidebar section interaction',
+											source: 'home page',
+										},
+									},
+								);
+							}
+						} }
+						onSearchEvent={ ( search, category ) => {
+							$e.run(
+								'kit-library/checkbox-filtration',
```

Comment:
The UI (checkbox) is not a part of the command
```suggestion
								'kit-library/filter',
```